### DISCUSSION TITLE
Ideias para o Layout Kit

### DISCUSSION THREAD

**Discussion by mustbetommy** [frontendbr/forum#18] [Discussão]
Fala pessoal,

Há algum tempo publiquei aqui o Layout Kit, um conjunto de arquivos e diretórios pré-configurados para desenvolver um layout simples sem ter que passar por todo o processo de organização.

O problema é o seguinte: eu estava pensando em deixar tudo ainda mais organizado. Por exemplo: adicionar os arquivos "package.json" e "gruntfile.js" básicos para compilação do "style.scss", entre outras modificações.

O que vocês acham? Tem mais alguma coisa que seria interessante adicionar/modificar?

Deem uma olhada no repo: https://github.com/kevinthomas-dev/layout-kit

**Comment by yurimatheus** [comment]
O link está fora.

_Cordialmente,_

_Yuri Gomes._

Em 3 de fevereiro de 2016 10:34, Kevin Thomas notifications@github.com
escreveu:

**Comment by mustbetommy** [comment]
@yurimatheus Pelo que vi na mensagem que foi enviada pra você, não tem ponto em "github.com

**Comment by yurimatheus** [comment]
Blz vi aqui... já tenho umas ideias em breve faço um pull.

_Cordialmente,_

_Yuri Gomes._

2016-02-03 11:54 GMT-03:00 Kevin Thomas notifications@github.com:

**Comment by rickbenetti** [comment]
Já fiz um fork, vou mexer aqui e daqui a pouco subo a sugestão.

**Comment by ribeiroevandro** [comment]
Eu estou fazendo algo assim, mas usando Jade para HTML e Stylus para CSS, em breve quero abrir ele aqui, estou ajustando umas coisas apenas por excesso de preciosismo...

**Comment by ribeiroevandro** [comment]
@kevinthomas-dev como dito acima, seu repositório não existe, pelo menos olhando seu perfil no github, não existe nenhum repositório público... Embora o @rkb81 disse ter feito um fork, vou tentar pelo perfil dele... Mas segue o print de quando tento acessar pelo o link informado.
![captura de tela 2016-02-05 as 00 41 10](https://cloud.githubusercontent.com/assets/943553/12836554/32a171fc-cba1-11e5-97a6-dbd4b0e5413c.png)
